### ISSUE TITLE
Not working after fresh install on MacOS

### ISSUE THREAD

**Issue by johanneskraem3r** [JetBrains/Unity3dRider#1]
I just tried this on a Mac after a brand new install, following your instructions, and got an issue. (Have been using this on Windows and it works beautifully.)

For some reason, it seems to open MonoDevelop for the solution files instead of opening them in Rider (Even tho Rider opens something, it fails in the end).

I think what happened is that I installed Rider before installing Unity and MonoDevelop, and so MonoDevelop somehow overwrote something like file association defaults for *.sln files, or something like that.

I will investigate, but I thought I'd leave a comment here either way.
(Thanks for the great work! I use Rider for Unity Dev by default now and I'm pretty happy with it!)

**Comment by johanneskraem3r** [comment]
Seems like the issue was solved by installing Mono on the mac :)

**Comment by kapyar** [comment]
Have the same issue but I have previously installed Monodevelop on my mac. How did u solve this problem?Just simply reinstall monodevelop?

**Comment by van800** [comment]
There should have been warning balloon in Rider saying that mono is not installed. Was it?

**Comment by johanneskraem3r** [comment]
There might have been but i missed it. I did then see a red Log Message

I focused mostly on the "load failed" Messages next to the assemblies, but
that was not very helpful.

Maybe there's a way to escalate the Error a bit more...

JK

On Aug 19, 2016 09:38, "Ivan Shakhov" <notifications@github.com> wrote:

There should have been warning balloon in Rider saying that mono is not
installed. Was it?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/JetBrains/Unity3dRider/issues/1#issuecomment-240949763>,
or mute the thread

**Comment by jan-ekholm** [comment]
I see the same for my tiny project. I used MonoDevelop up until yesterday and then installed a EAP version of Rider. I saw no errors referring to Mono, just some that Rider had crashed internally. :) Editing still kind of works and I can select files from the breadcrumb thing at the top of the window and switch between open files with C-Tab. It was however not a too impressive first launch experience. Reformatting code does not work either, but in general editing is already way better than MonoDevelop. My last startup error was filed as https://youtrack.jetbrains.com/oauth?state=%2Fissue%2FDEXP-160579

![screen shot 2017-01-02 at 15 56 26](https://cloud.githubusercontent.com/assets/1058619/21601902/e05afb4a-d197-11e6-9519-b98061d41ed2.png)
